reporting_app/static/column_formatter.js

```diff
@@ -77,8 +77,8 @@ function string_formatter(cell_data, fmt, row){
 
     }
 
-    // if the list is longer than 1, then it should be rendered as a dropdown
-    if (formatted_data.length > 1) {
+    // if the list is longer than 1 or if it has special formatting, then it should be rendered as a dropdown
+    if (formatted_data.length > 1 || fmt['link_format_function']) {
```

Comment:
The "special formatting" refers to the the function which displays the length of the array and not the contents of it, in designated places.